src/lib/Libwin/winnet.c

```diff
@@ -54,15 +54,14 @@
 int
 winsock_init()
 {	
-	char	logb[LOG_BUF_SIZE] = {'\0' } ;
 	WSADATA	data;
 
 	save_env(); 	/* need certain environment variables set in order */
 	/* make network calls like socket() or gethostbyname()*/
 
 	if (WSAStartup(MAKEWORD(2, 2), &data)) {
-		sprintf(logb,"winsock_init failed! error=%d", WSAGetLastError());
-		log_err(-1, "winsock_init", logb);
+		log_eventf(PBSEVENT_ERROR, 0, LOG_DEBUG, __func__, "winsock_init failed with errno %d",
+				WSAGetLastError());
```

Comment:
please print only the return code of WSAStartup. from https://docs.microsoft.com/en-us/windows/win32/api/winsock/nf-winsock-wsastartup

> The WSAStartup function directly returns the extended error code in the return value for this function. A call to the WSAGetLastError function is not needed and should not be used.